src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -1561,6 +1581,25 @@ function highlightRubricCol(index, questionNumber, highlight) {
 }
 
 /**
+ * Attaches event handlers to "assign weights" checkboxes to toggle the visibility
+ * of the input boxes for rubric weights
+ */
+function bindAssignWeightsCheckboxes() {
+    $('body').on('click', 'input[id^="rubricAssignWeights"]', function(e) {
+        $(this).closest('form').find('input[id^="rubricWeight"]').toggle();
+    });
+}
+
+/**
+ * @param questionNumber
+ *            the question number of the feedback question
+ * @returns {Boolean} true if the weights are assigned by the user, otherwise false
+ */
+function hasAssignedWeights(questionNumber) {
```

Comment:
Yes, it will break things here. Most of the existing logic and behaviour in this file relies on question numbers though, we should make a new issue to change the existing logic in this file and in the InstructorFeedbackEditPageData to use the questionId instead.